### PR TITLE
Update contribute.md

### PR DESCRIPTION
**Pull request by pavlenex**
Co-Authored-By: @sqcken
#220 #222

### CODE REVIEW

**Comment by pavlenex** [comment]
I approve this since it touches the `Contribute` section, but since I opened a PR, I need someone else to formally approve it  so we can get it merged in✅ 😄

File: guide/contribute/contribute.md [hunk 1]
```diff
@@ -61,14 +60,14 @@ Tips on how to write for the guide. The goal is to achieve a consistent tone acr
...
 
-Notes on how to prepare visuals for use in the guide. We encourage diverse creative expressions, so this page focuses mostly on basics like file names and image sizes.
+Creating visuals for the Bitcoin Design Guide? We recommend following these best practices for file naming, image sizing, and on.
 
 ---
 
 **[Formatting]({{ '/guide/contribute/formatting/' | relative_url }})**
 
-Layout and formatting options available for content pages, from headers and blockquotes to images and embeds.
+Layout and formatting recommendations from headers and blockquotes to images and embeds.
```
Comment (GBKS) [inline guide/contribute/contribute.md:69]: `recommendations` isn't quite right, as the page lists what's technically possible. It's more of a technical documentation page.

Comment (pavlenex) [reply guide/contribute/contribute.md:69]: Nice catch, will change it to `options`.

Comment (pavlenex) [reply guide/contribute/contribute.md]: https://github.com/BitcoinDesign/Guide/pull/224/commits/f6e588140138147c26ecf4f38994d4b1eab80dc5

**Review by GBKS — APPROVED** [review]

**Pull request merged**